SimG4Core/CustomPhysics/src/CMSSQNeutronAnnih.cc

```diff
@@ -87,7 +87,7 @@ G4double CMSSQNeutronAnnih::momDistr(G4double x_in) {
 
   //now interpolate the above points for x_in
   G4double result = 9999;
-  for (int i = 0; i < n_entries; i++) {
+  for (int i = 1; i < n_entries; i++) {
```

Comment:
clang-analyzer was complaining that for `i=0`, we are trying to access `CDF_k[-1] and x[-1`.  So I started the loop from `1`. @cms-sw/simulation-l2 please check if this is correct

Reply:
I am not sure what is real range for x_in but I would use following code, which is valid in all cases:

  if (x_in <= 0.0) { return 0.0; }
  if (x_in >= 1.0) { return CDF_k[n_entries - 1];}
  for (int i = 1; i < n_entries; i++) {
    if (x[i] >= x_in) {
      return (CDF_k[i] - CDF_k[i - 1]) / (x[i] - x[i - 1]) * (x_in - x[i - 1]) + CDF_k[i - 1];
    }
  }
  return 0.0;

Reply:
done